tests/unit/s2n_session_ticket_test.c

```diff
@@ -372,36 +372,35 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_config_set_wall_clock(server_config, mock_time, &mock_current_time));
 
         /* Add one ST key */
-        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name1, s2n_array_len(ticket_key_name1), ticket_key1, s2n_array_len(ticket_key1), 0));
+        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name2, s2n_array_len(ticket_key_name2), ticket_key2, s2n_array_len(ticket_key2), 0));
 
         /* Add a mock delay such that key 1 moves to decrypt-only state */
         mock_current_time += server_config->encrypt_decrypt_key_lifetime_in_nanos;
 
         uint32_t key_intro_time = mock_current_time / ONE_SEC_IN_NANOS;
-        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name2, s2n_array_len(ticket_key_name2),
-                ticket_key2, s2n_array_len(ticket_key2), key_intro_time));
+        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name1, s2n_array_len(ticket_key_name1),
+                ticket_key1, s2n_array_len(ticket_key1), key_intro_time));
+
+        /* Verify there is an encrypt key available */
+        s2n_result_is_ok(s2n_config_is_encrypt_key_available(server_config));
```

Comment:
`s2n_result_is_ok` returns whether an error occurred, but there's nothing here to assert that the bool is true. `EXPECT_OK` does this:
https://github.com/aws/s2n-tls/blob/ebaf52283949a2208d1f80fceabc4aefc7254973/tests/s2n_test.h#L203
```suggestion
        EXPECT_OK(s2n_config_is_encrypt_key_available(server_config));
```